test/fixture/config/agent_bad_agent_block.conf

```diff
@@ -0,0 +1,4 @@
+agent {
+    unknwon_option1 = "unknwon_option1"
```

Comment:
nit: typo on `unknwon` (copied and pasted throughout the text fixtures)

Reply:
I fixed it with https://github.com/spiffe/spire/pull/1289/commits/fab9ef08e460fca3b1be00c6809ec0e120ae6354 !